core/unit_test/tools/printing-tool.cpp

```diff
@@ -47,11 +49,12 @@ extern "C" void kokkosp_begin_parallel_reduce(const char* name,
                                               const uint32_t devID,
                                               uint64_t* kID) {
   *kID = parallel_reduce_id;
-  printf("kokkosp_begin_parallel_reduce:%s:%u:%lu::", name, devID, *kID);
+  printf("kokkosp_begin_parallel_reduce:%s:%u:%lu::", name, devID,
+         static_cast<unsigned long>(*kID));
 }
 
 extern "C" void kokkosp_end_parallel_reduce(const uint64_t kID) {
-  printf("kokkosp_end_parallel_reduce:%lu::", kID);
+  printf("kokkosp_end_parallel_reduce:%lu::", static_cast<unsigned long>(kID));
```

Comment:
I assume the static cast is to get rid of a warning? Again, question is PRI codes.